src/index.js

```diff
@@ -27,9 +27,10 @@ module.exports = ({ types: t }) => {
         if (t.isIdentifier(node.tag, { name: "gql" })) {
           for (let element of node.quasi.quasis) {
             element.value.raw = minifyRaw(element.value.raw);
+            element.value.cooked = minifyRaw(element.value.cooked);
```

Comment:
Could you change the method name since it no longer just minifes the `raw` version? I don't even know what the "cooked" version is so I guess this change is fine in general.

Reply:
> I don't even know what the "cooked" version is

(same!) 😄

Reply:
I've pushed a change @Jalle19, thanks.